pkg/genlib/testdata/template.tpl

```diff
@@ -0,0 +1 @@
+{{generate "aws.vpcflow.version"}} {{generate "aws.vpcflow.account_id"}} {{generate "aws.vpcflow.interface_id"}} {{generate "aws.vpcflow.pkt_srcaddr"}} {{generate "aws.vpcflow.pkt_dstaddr"}} {{generate "source.port"}} {{generate "destination.port"}} {{generate "network.transport"}} {{generate "network.packets"}} {{generate "network.bytes"}} {{generate "event.start"}} {{generate "event.end"}} {{generate "aws.vpcflow.action"}} {{generate "aws.vpcflow.log_status"}}
```

Comment:
can we add the logic for packets and log status here?
in order to compare the benchmark we should generate the same data as in https://github.com/elastic/elastic-integration-corpus-generator-tool/blob/047ef7f2f5b59b7a918c283f26e8622309e5c0e4/pkg/genlib/generator_test.go#L805